### PR TITLE
FIX acc_invoice_pricelist: recompute tax after update pricelist

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

File: account_invoice_pricelist/models/account_invoice.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def _onchange_partner_id_account_invoice_pricelist(self):
     def button_update_prices_from_pricelist(self):
         for inv in self.filtered(lambda r: r.state == 'draft'):
             inv.invoice_line_ids.filtered('product_id').update_from_pricelist()
+        self.filtered(lambda r: r.state == 'draft').compute_taxes()
```
Comment (bealdav) [inline account_invoice_pricelist/models/account_invoice.py:28]: Without this line taxes are wrong.
In production since one week
ping @legalsylvain @rvalyi

**Comment by bealdav** [comment]
ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Review by PierrickBrun — APPROVED** [review]

**Comment by legalsylvain** [comment]
thanks @bealdav.

**Pull request closed without merging**